src/app/dashboard/dashboard.service.test.ts

```diff
@@ -42,6 +43,23 @@ describe('DashboardService', () => {
         });
     });
 
+    describe('#getStatistics', () => {
+        let deviceId, period;
+
+        beforeEach(() => {
+            deviceId = Math.random();
```

Comment:
This is called [property testing](https://en.wikipedia.org/wiki/Property_testing)